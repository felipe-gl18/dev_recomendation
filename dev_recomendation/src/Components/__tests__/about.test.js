import React from 'react';
import renderer from 'react-test-renderer';
import About from '../aboutComponent';

it('snapshot about', () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
})