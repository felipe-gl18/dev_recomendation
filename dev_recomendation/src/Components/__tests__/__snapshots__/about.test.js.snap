// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot about 1`] = `
<div
  className="aboutme"
>
  <div
    className="aboutme-content"
  >
    <h1
      className="aboutmeName"
    >
      Felipe Gadelha
    </h1>
    <div
      className="middle"
    >
      <p
        className="aboutmeDescription"
      >
        Sou um programador Full Stack Júnior, e construí esse site para
        <br />
        botar em prática tudo que aprendi sobre Node.js, mas pretendo
        <br />
        dar continuação ao projeto, até porque, saber por onde começar
        <br />
        foi uma das minhas grandes dificuldades quando iniciei meus
        <br />
        estudos na área. Caso queira entrar em contato, só escolher uma
        <br />
        das opções abaixo.
      </p>
      <div
        className="profile"
      />
    </div>
    <div
      className="div-form-text-social"
    >
      <a
        href="https://www.instagram.com/felipe.linog/"
      >
        <i
          className="fab fa-instagram"
        />
      </a>
      <a
        href="https://www.linkedin.com/in/felipe-lino-273578207/"
      >
        <i
          className="fab fa-linkedin"
        />
      </a>
      <a
        href=" https://wa.me/5588992048450/"
      >
        <i
          className="fab fa-whatsapp-square"
        />
      </a>
    </div>
  </div>
</div>
`;
